wpeverest/everest-forms
# Fix - Deleted Field Entries are shown in Single Entry

### All Submissions:






### Changes proposed in this Pull Request:

Previously, the **Field Name** was not showing in **Entries** that were removed from the **Forms**. This PR helps to view **Field Name** that is Deleted from the Form.

### How to test the changes in this Pull Request:

1. Go to the **Entries** which have deleted fields.
2. Click on **View** Button to see the Entry.
3. That's all. Now you can see Field name as well instead of **Field ID** which was previously showing when the field is deleted from the Form.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Deleted Field Entries are shown in Single Entry.
